### DISCUSSION TITLE
Multiple destination support

### DISCUSSION THREAD

**Discussion by pulse-mind** [Archmonger/django-dbbackup#569] [Feature Requests]
For example, I'd like to store the backup on the local server and also to AWS3. 
Is it possible to configure DBBackup in this way ? 
I would like to avoid running two times django_dbbackup with two configurations.

**Comment by ZuluPro** [comment]
Hi @pulse-mind 
I guess the issue #193 could fill your problem.

**Comment by diego-lutke** [comment]
I don't think signals will address @pulse-mind 's issue. Please correct me if I am wrong.

I am interested in this issue as well, specifically on how to configure multiple locations. I think the #357 PR addresses it; looks like it wasn't merged though.

**Comment by pulse-mind** [comment]
I was thinking the same about signals.

**Comment by Archmonger** [comment]
#357 was not merged due to merge conflicts and the author not having time to resolve them.

I am open to any branches that work off that PR.

**Comment by Archmonger** [comment]
Closing since this is a duplicate of #580
